website/src/get-started/wordcount-example.md

```diff
@@ -1362,7 +1379,20 @@ PCollection<KV<String, Long>> wordCounts = windowedWords.apply(new WordCount.Cou
 ```
 
 ```py
-# This feature is not yet available in the Beam SDK for Python.
+class CountWordsFn(Dofn):
+   def __init__(self):
```

Comment:
I think we can drop the counter and __init__ for this snippet. (and `self.words_counter.inc(len(words))  below.